<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mural de Vagas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Código da vaga:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="codigoVaga">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-2</number>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLowercaseOnly</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string extracomment="Favor digitar o código da vaga pretendida"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCandidatarMural">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 255, 245);
color: rgb(3, 28, 255);</string>
    </property>
    <property name="text">
     <string>Candidatar-se</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Digite abaixo o código da vaga pretendida</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>721</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string> Candidato confirmar email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailCandidato">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mensagemDePreenchimento">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbVolarMuralVagas">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 255, 245);
color: rgb(3, 28, 255);</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
